src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+
+CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';
+
+USE airlines;
+
+DROP TABLE IF EXISTS `airplanes`;
+DROP TABLE IF EXISTS `crews_crew_members`;
+DROP TABLE IF EXISTS `crews`;
+DROP TABLE IF EXISTS `crew_members`;
+
+CREATE TABLE `crews` (
+                         `id` INT AUTO_INCREMENT PRIMARY KEY,
+                         `name` VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE `airplanes` (
+                             `id` INT AUTO_INCREMENT PRIMARY KEY,
+                             `code name` VARCHAR(10) NOT NULL,
+                             `model` VARCHAR(30) NOT NULL,
+                             `manufacture date` DATE NOT NULL,
```

Comment:
in SQL database columns in most cases follows underscore as delimiter:
```
code_name VARCHAR(10) NOT NULL,
...
manufacture_date DATE NOT NULL,
```

Reply:
fixed in